### ISSUE TITLE
Connection with 2FA Microsoft not possible

### ISSUE THREAD

**Issue by BenJ3D** [IsmaelMartinez/teams-for-linux#1530]
**Describe the bug**
Impossible to connect with the 2FA active on my microsoft account, no window opens to suggest me to use my password, or my PIN+Yubikey etc, I remain blocked on the message:

Face, fingerprint, PIN or security key
Your device will open a security window. Follow the instructions to log in.

**To Reproduce**
Log in with the 2FA on our activated micropsoft account (I also have the Yubikey, I don't know if this changes anything, because normally I can always log in with my long PIN or account password).

**Expected behavior**
Be able to enter your password or use your PIN/physical key for the 2FA.

**Screenshots**
![image](https://github.com/user-attachments/assets/3b6ed119-f379-454e-b6d4-5d759e61e04a)

**Desktop (please complete the following information):**
 - OS: Ubuntu 24.04
 - Installation package : Ubuntu software UI ?!
![image](https://github.com/user-attachments/assets/a76c9cd5-827b-4f3c-b8c6-3cffd70e9995)
 - Version 1.12.3

**Debug**
When possible, please run the application from the terminal using `--webDebug` and try to reproduce the error.

The provide in this section the output from both the terminal and the browser debug console.

```bash
teams-for-linux --webDebug --logConfig='{}'
```

```
➜  ~ teams-for-linux --webDebug --logConfig='{}'
No config file found, using default values
all good with appLogLevels you aren't using them
Initialising logger with config: "{}"
10:43:14.724 › File logging at /home/<myname>/snap/teams-for-linux/730/.config/teams-for-linux/logs/main.log with level silly
10:43:14.725 › Console logging enabled with level silly
10:43:14.726 › Logger initialised
10:43:14.726 › configPath: /home/<myname>/snap/teams-for-linux/730/.config/teams-for-linux
10:43:14.726 › configFile: {}
dbus-send: /snap/teams-for-linux/730/lib/x86_64-linux-gnu/libdbus-1.so.3: version `LIBDBUS_PRIVATE_1.12.20' not found (required by dbus-send)

(teams-for-linux:115428): Gtk-WARNING **: 10:43:14.778: Theme parsing error: gtk.css:1422:23: 'font-feature-settings' is not a valid property name

(teams-for-linux:115428): Gtk-WARNING **: 10:43:14.781: Theme parsing error: gtk.css:3308:25: 'font-feature-settings' is not a valid property name

(teams-for-linux:115428): Gtk-WARNING **: 10:43:14.782: Theme parsing error: gtk.css:3770:23: 'font-feature-settings' is not a valid property name
[115428:1219/104314.826037:ERROR:object_proxy.cc(576)] Failed to call method: org.freedesktop.Secret.Service.ReadAlias: object_path= /org/freedesktop/secrets: org.freedesktop.DBus.Error.AccessDenied: An AppArmor policy prevents this sender from sending this message to this recipient; type="method_call", sender=":1.288" (uid=1000 pid=115428 comm="/snap/teams-for-linux/730/teams-for-linux --no-san" label="snap.teams-for-linux.teams-for-linux (enforce)") interface="org.freedesktop.Secret.Service" member="ReadAlias" error name="(unset)" requested_reply="0" destination="org.freedesktop.secrets" (uid=1000 pid=3953 comm="/usr/bin/gnome-keyring-daemon --foreground --compo" label="unconfined")
10:43:14.853 › processArgs: [
  '/snap/teams-for-linux/730/teams-for-linux',
  '--no-sandbox',
  '--webDebug',
  '--logConfig={}'
]
10:43:14.854 › testing RegExp processArgs false
10:43:14.854 › testing RegExp processArgs false
10:43:14.854 › testing RegExp processArgs false
10:43:14.854 › testing RegExp processArgs false
10:43:14.855 › Waiting for network...
10:43:14.855 › Testing network using net.request() for https://teams.microsoft.com
10:43:14.856 › Spellchecker is disabled!

(teams-for-linux:115428): Gtk-WARNING **: 10:43:14.866: Theme parsing error: gtk.css:1422:23: 'font-feature-settings' is not a valid property name

(teams-for-linux:115428): Gtk-WARNING **: 10:43:14.868: Theme parsing error: gtk.css:3308:25: 'font-feature-settings' is not a valid property name

(teams-for-linux:115428): Gtk-WARNING **: 10:43:14.869: Theme parsing error: gtk.css:3770:23: 'font-feature-settings' is not a valid property name
[115428:1219/104314.929035:ERROR:object_proxy.cc(576)] Failed to call method: org.freedesktop.login1.Manager.Inhibit: object_path= /org/freedesktop/login1: org.freedesktop.DBus.Error.AccessDenied: An AppArmor policy prevents this sender from sending this message to this recipient; type="method_call", sender=":1.389" (uid=1000 pid=115428 comm="/snap/teams-for-linux/730/teams-for-linux --no-san" label="snap.teams-for-linux.teams-for-linux (enforce)") interface="org.freedesktop.login1.Manager" member="Inhibit" error name="(unset)" requested_reply="0" destination="org.freedesktop.login1" (uid=0 pid=1813 comm="/usr/lib/systemd/systemd-logind" label="unconfined")
10:43:15.005 › Network test successful with method https
[115428:1219/104315.272614:ERROR:CONSOLE(1)] "Unknown VE context: language-mismatch    at new He (devtools://devtools/bundled/ui/legacy/legacy.js:1:98356)
    at devtools://devtools/bundled/ui/legacy/legacy.js:1:191793
    at new oi (devtools://devtools/bundled/ui/legacy/legacy.js:1:192189)
    at oi.instance (devtools://devtools/bundled/ui/legacy/legacy.js:1:192311)
    at O.presentUI (devtools://devtools/bundled/panels/emulation/emulation.js:1:58454)
    at #I (devtools://devtools/bundled/entrypoints/main/main.js:1:17314)
    at #x (devtools://devtools/bundled/entrypoints/main/main.js:1:17109)", source: devtools://devtools/bundled/ui/visual_logging/visual_logging.js (1)
10:43:15.284 › GetSystemIdleState => IdleTimeout: 300s, IdleTimeoutPollInterval: 10s, ActiveCheckPollInterval: 2s, IdleTime: 0s, IdleState: 'active'
[115428:1219/104315.285704:ERROR:object_proxy.cc(576)] Failed to call method: org.freedesktop.ScreenSaver.GetActive: object_path= /org/freedesktop/ScreenSaver: org.freedesktop.DBus.Error.NotSupported: This method is not part of the idle inhibition specification: https://specifications.freedesktop.org/idle-inhibit-spec/latest/
[115428:1219/104315.415413:ERROR:CONSOLE(1)] "Request Autofill.enable failed. {"code":-32601,"message":"'Autofill.enable' wasn't found"}", source: devtools://devtools/bundled/core/protocol_client/protocol_client.js (1)
[115428:1219/104315.415447:ERROR:CONSOLE(1)] "Request Autofill.setAddresses failed. {"code":-32601,"message":"'Autofill.setAddresses' wasn't found"}", source: devtools://devtools/bundled/core/protocol_client/protocol_client.js (1)
10:43:15.694 › did-frame-finish-load {} true
10:43:15.695 › did-finish-load
10:43:16.533 › did-frame-finish-load {} false
10:43:16.860 › Badge count set to '0'
10:43:16.870 › Badge count set to '0'
10:43:17.192 › assignOnDidFailLoadEventHandler : {} - -3 - 
10:43:17.655 › did-frame-finish-load {} false
10:43:17.828 › Badge count set to '0'
10:43:17.928 › assignOnDidFailLoadEventHandler : {} - -3 - 
10:43:19.054 › did-frame-finish-load {} false
10:43:19.444 › assignOnDidFailLoadEventHandler : {} - -3 - 
[115428:1219/104319.481007:ERROR:CONSOLE(1)] "Request Autofill.enable failed. {"code":-32601,"message":"'Autofill.enable' wasn't found"}", source: devtools://devtools/bundled/core/protocol_client/protocol_client.js (1)
[115428:1219/104319.481080:ERROR:CONSOLE(1)] "Request Autofill.setAddresses failed. {"code":-32601,"message":"'Autofill.setAddresses' wasn't found"}", source: devtools://devtools/bundled/core/protocol_client/protocol_client.js (1)
10:43:19.486 › did-frame-finish-load {} false
10:43:21.227 › did-frame-finish-load {} false
10:43:21.492 › assignOnDidFailLoadEventHandler : {} - -3 - 
10:43:21.798 › GetSystemIdleState => IdleTimeout: 300s, IdleTimeoutPollInterval: 10s, ActiveCheckPollInterval: 2s, IdleTime: 0s, IdleState: 'active'
10:43:21.856 › did-frame-finish-load {} true
10:43:21.856 › did-finish-load
10:43:22.748 › GetSystemIdleState => IdleTimeout: 300s, IdleTimeoutPollInterval: 10s, ActiveCheckPollInterval: 2s, IdleTime: 0s, IdleState: 'active'
10:43:22.824 › did-frame-finish-load {} true
10:43:22.824 › did-finish-load
[115428:1219/104322.947477:ERROR:udev_watcher.cc(97)] Failed to begin udev enumeration.
10:43:32.788 › GetSystemIdleState => IdleTimeout: 300s, IdleTimeoutPollInterval: 10s, ActiveCheckPollInterval: 2s, IdleTime: 0s, IdleState: 'active'


```

**Comment by IsmaelMartinez** [comment]
Unfortunately electron doesn't support Yubikeys. 

See:
https://github.com/IsmaelMartinez/teams-for-linux/issues/1517
https://github.com/IsmaelMartinez/teams-for-linux/issues/1433
https://github.com/IsmaelMartinez/teams-for-linux/issues/1299
https://github.com/IsmaelMartinez/teams-for-linux/issues/1166
https://github.com/IsmaelMartinez/teams-for-linux/issues/1126

**Comment by jreuschPSI** [comment]
TWIMC: Seems this is the upstream issue to follow https://github.com/electron/electron/issues/24573
